templates/base.html

```diff
@@ -38,7 +38,8 @@
     <link rel="icon" type="image/png" href="{{ url_for('static', filename='img/favicon/favicon-16x16.png') }}" sizes="16x16">
     <link rel="manifest" href="{{ url_for('static', filename='img/favicon/manifest.json') }}">
     <link rel="mask-icon" href="{{ url_for('static', filename='img/favicon/safari-pinned-tab.svg') }}" color="#5bbad5">
-    <link rel="shortcut icon" href="{{ url_for('static', filename='img/favicon/favicon.ico') }}">
+    <!-- favicon changes depending on server type -->
+    <link rel="shortcut icon" href="{{ url_for('static', filename=config.FAVICON.get(config.SERVER_TYPE))  }}">
```

Comment:
with no config entry found, code can still load default favicon icon `img/favicon/favicon.ico`

Reply:
Since config entry is checked in core.py, default will be passed to favicon from there.